lib/parlour/plugin.rb

```diff
@@ -28,7 +28,7 @@ def self.inherited(new_plugin)
       registered_plugins[T.must(new_plugin.name)] = new_plugin
     end
 
-    sig { params(plugins: T::Array[Plugin], generator: RbiGenerator).void }
+    sig { params(plugins: T::Array[Plugin], generator: RbiGenerator, allow_failure: T::Boolean).void }
```

Comment:
@AaronC81 we probably should set up git push hook to run `srb tc` and disallow any error committed. 
Is this file tested? If this file is tested, why doesn't this cause a sorbet run time error :-?